xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -322,8 +322,8 @@ In the case of <xref:System.IO.Stream>, this writes the data to the stream and f
         <ReturnType>System.Text.Json.JsonWriterOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the custom behavior when writing JSON using this instance, which indicates whether to format the output while writing and whether to skip structural JSON validation.</summary>
-        <value>An object that defines the behavior of this instance for formatting and validation.</value>
+        <summary>Gets the custom behavior when writing JSON using this instance, which indicates whether to format the output while writing, whether to skip structural JSON validation, and which characters to escape.</summary>
+        <value>Defines the behavior of this instance of the writer for formatting, validation, and escaping.</value>
```

Comment:
JsonWriterOptions is a struct, not a class.

Reply:
It's my understanding that `<value>` needs to explain the data type and what it represents. So you could prepend it with "struct instance" for example. Also, as I mentioned in your other PR, in our official docs for "C# objects", it seems that we use "object" to describe a struct instance too.

Reply:
Maybe something like this? It shouldn't start with a verb here. 

```suggestion
        <value>The custom behavior of this instance of the writer for formatting, validating, and escaping.</value>
```